models/players.js

```diff
@@ -77,34 +79,35 @@ switch (league){
                case 'nba':
                json_response = JSON.parse(body);
                players_sorted = sortNBA(json_response);
-               players = formatPlayers(players_sorted, team_id);
-               mongoInsertPlayers(team_id, players);
-               callback(players.players, res, league)
+               players = formatPlayers(players_sorted, rb_team_id);
+               mongoInsertPlayers(rb_team_id, players);
+               callback(players.players, rb_team_id, res, league)
                break;
                case 'nfl':
                json_response = JSON.parse(body);
                players_sorted = sortNFL(json_response);
-               players = formatPlayers(players_sorted, team_id);
-               mongoInsertPlayers(team_id, players);
-               callback(players.players, res, league)
+               players = formatPlayers(players_sorted, rb_team_id);
+               mongoInsertPlayers(rb_team_id, players);
+               callback(players.players, rb_team_id, res, league)
                break;
                case 'nhl':
                 json_response = JSON.parse(body);
-                players = formatPlayers(json_response, team_id);
-                mongoInsertPlayers(team_id, players);
-                callback(players.players, res, league)
+                players = formatPlayers(json_response, rb_team_id);
+                mongoInsertPlayers(rb_team_id, players);
+                callback(players.players, rb_team_id, res, league)
                 break;
                case 'eu_soccer':
                 playersParsed = formatEUSoccerPlayers(response.body);
-                players = formatPlayersDocument(team_id, playersParsed);
-                mongoInsertPlayers(team_id, players);
-                callback(players.players, res, league)
+                players = formatPlayersDocument(rb_team_id, playersParsed);
+                mongoInsertPlayers(rb_team_id, players);
+                callback(players.players, rb_team_id, res, league)
                 break;
                case 'mlb':  
+               // THIS ENCRYPTION ACTIVITY WILL NEED TO BE DIFFERENT
                 playersParsed = formatMLBPlayers(response.body, team_id);
                 players = formatPlayersDocument(team_id, playersParsed);
                 mongoInsertPlayers(team_id, players);
-                callback(players.players, res, league)
+                callback(players.players, rb_team_id, res, league)
                 break;
```

Comment:
one other thing i noticed. All of the leagues look like they share the final two function (mongoInsertPlayers and callback.) These can be moved out of the case statement.